samples/c2pa/native-dash/c2pa-demo-native.html

```diff
@@ -35,17 +39,19 @@
         function init() {
             var video,
                 dashPlayer,
-                url = "../../microsoft/video_samples/DASH/audio_video_sample/manifest.mpd";
+                url = "../../bunny_out/BigBuckBunny_2s_simple_2014_05_09.mpd";
```

Comment:
Can you change this to use the public URL?

Reply:
brought it in and works as expected